lib/src/main/kotlin/dev/chrisbanes/snapper/LazyList.kt

```diff
@@ -62,6 +67,7 @@ public fun rememberSnapperFlingBehavior(
     endContentPadding: Dp = 0.dp,
     decayAnimationSpec: DecayAnimationSpec<Float> = rememberSplineBasedDecay(),
     springAnimationSpec: AnimationSpec<Float> = SnapperFlingBehaviorDefaults.SpringAnimationSpec,
+    snapIndex: ((SnapperLayoutInfo, targetIndex: Int) -> Int)? = null,
```

Comment:
maybe it doesn't make sure to make this lambda nullable, we introduce extra condition, and you will have to explain in the docs what null does mean. maybe instead you just provide a default implementation which returns targetIndex

Reply:
Done